### PR TITLE
Mobile version: Works and about sections

### PR DESCRIPTION
**Pull request by bekjon-me**
- created assets folder and moved asset files(images, styles) there
- added works section
- added about section

### CODE REVIEW

File: src/assets/style.css [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+  cursor: pointer;
+}
+
+section#works > .cards > .card > button:active {
+  background: #e05304;
+  border: 1px solid transparent;
+  color: #f4f1f0;
+  cursor: pointer;
+}
+
```
Comment (wayungi) [inline src/assets/style.css]: - [ OPTIONAL] Please do not be too specific with your selectors. Be specific enough to target the elements you want. over specificity, increase processing overhead and makes the page slow.
- [ OPTIONAL] Please write DRY CSS. You create one class ie `.card` and assign it the display property of flex.  This class can then be assigned to all your individual work `divs`

File: src/assets/style.css [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+/* WORKS SECTION ENDED */
+
+/* ABOUT SECTION STARTED */
+
+section#about {
+  padding-top: 120px;
+  padding-bottom: 120px;
+  background: #1c1a19;
+}
```
Comment (wayungi) [inline src/assets/style.css:334]: - [ ] Please align the content inside this `section#about` selector with flexbox. You have instead aligned only the about-me div with flex.

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @bekjon-me , 

Very very good job so far. !
There is just one issue that you still need to work on to go to the next project but you are almost there!
image


## Highlights
- [ ] Good PR title 👏🏾 
- [ ] GitHub flow 👏🏾 
- [ ] Descriptive commit message 👏🏾 
- [ ] Professional README 👏🏾 
- [ ] All linter checks passing 👏🏾 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @bekjon-me, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**